app/Http/Controllers/CampaignRegistrationController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Campaign;
+use App\Models\User;
+use App\Models\Character;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
+
+class CampaignRegistrationController extends Controller
+{
+    /**
+     * @param \Illuminate\Http\Request $request
+     * @return \Inertia\Response
+     */
+    public function create(Request $request)
+    {
+        $characters = Auth::user()->characters;
+
+        $code = $request->validate([
+            'code' => "sometimes|string"
+        ])['code']?? null;
```

Comment:
PSR-1 coding standards

```suggestion
        ])['code'] ?? null;
```